code/datums/elements/food/processable.dm

```diff
@@ -54,3 +61,24 @@
 		examine_list += span_notice("It can be turned into [amount_created] [initial(result_atom_type.name)]s with <b>[tool_behaviour_name(tool_behaviour)]</b>!")
 	else
 		examine_list += span_notice("It can be turned into \a [initial(result_atom_type.name)] with <b>[tool_behaviour_name(tool_behaviour)]</b>!")
+
+/**
+ * Adds context sensitivy directly to the processable file for screentips
+ * Arguments:
+ * * source - refers to item that will display its screentip
+ * * context - refers to, in this case, an item that can be proccessed into another item via add element proccessable
+ * * held_item - refers to tool used by add element proccessable to process one item into another item
+ * * user - refers to user who will see the screentip when the proper context and tool are there
+ */
+/datum/element/processable/proc/on_requesting_context_from_item(datum/source, list/context, obj/item/held_item, mob/user,)
```

Comment:
```suggestion
/datum/element/processable/proc/on_requesting_context_from_item(datum/source, list/context, obj/item/held_item, mob/user)
```

Cruft when you flattened it into one line. won't affect compile but it looks bad